docs/configure/agent.md

```diff
@@ -0,0 +1,313 @@
+{{< /alert >}}
+
+If `roaming_mode` is enabled, `agent-provisioning` will try to connect to each specified network in order of `priority` from highest to lowest.
+If the highest-priority network is not available or the machine can connect but internet is not available, `viam-agent` will then attempt to connect to the next-highest network, and so on until all configured networks have been tried.
+
+<!-- prettier-ignore -->
+| Name       | Type   | Description |
+| ---------- | ------ | ----------- |
+| `type`     | string | The type of the network. Options: `"wifi"`|
+| `ssid`     | string | The network's SSID. |
+| `psk`      | string | The network pass key. |
+| `priority` | int    | Priority to choose the network with. Values between -999 and 999. Default: `0`. |
+
+The following configuration defines the connection information and credentials for two WiFi networks named `fallbackNetOne` and `fallbackNetTwo`:
+
+```json {class="line-numbers linkable-line-numbers"}
+...
+"agent": {
+  "agent-provisioning": {
+    ...
+    "attributes": {
+      ...
+      "roaming_mode": true,
+      "networks": [
+        {
+          "type": "wifi",
+          "ssid": "fallbackNetOne",
+          "psk": "myFirstPassword",
+          "priority": 30
+        },
+        {
+          "type": "wifi",
+          "ssid": "fallbackNetTwo",
+          "psk": "mySecondPassword",
+          "priority": 10
+        }
+      ]
+    }
+  }
+}
+```
+
+### `agent-syscfg`
+
+<!-- prettier-ignore -->
+| Option | Type | Required? | Description |
+| ------ | ---- | --------- | ----------- |
+| `attributes` | object | Optional | <ul><li>`logging`: parameters for logging<ul>`disable`: when set to `true`, removes any prior tweaks to the logging config and disables the use of defaults??.<li>`system_max_use`: sets the maximum disk space journald will user for persistent log storage. Numeric values are in bytes, with optional single letter suffix for larger units, for example. K, M, or G. Default: `512M`.</li><li>`runtime_max_use`: sets the runtime/temporary limit. Numeric values are in bytes, with optional single letter suffix for larger units, for example. K, M, or G. Default: `512M`.</li></ul></li><li>`upgrades`: Using `upgrades` installs the `unattended-upgrades` package, and replace `20auto-upgrades` and `50unattended-upgrades` in <FILE>/etc/apt/apt.conf.d/</FILE>, with the latter's Origins-Pattern list being generated automatically from configured repositories on the system, so custom repos (at the time the setting is enabled) will be included.<ul><li>`type`: Configured unattended upgrades for Debian bullseye and bookworm. Options: `""` (no effect), `"disable"` (disables automatic upgrades), `"security"` (only enables updates from sources with security in their codename, ex: bookworm-security), `"all"` (enable updates from all configured sources).</li></ul></li></ul> |
```

Comment:
@Otterverse I don't understand the disable option. It disables the use of defaults? What does that mean?